addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,42 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("can reply to logged note in chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    const channelId = pyEnv["discuss.channel"].create({ name: "general" });
+    pyEnv["mail.message"].create([
+        {
+            author_id: partnerBId,
+            body: "Test message from B",
+            model: "res.partner",
+            res_id: serverState.partnerId,
+            subtype_id: pyEnv["mail.message.subtype"].search([
+                ["subtype_xmlid", "=", "mail.mt_note"],
+            ])[0],
+        },
+        {
+            author_id: serverState.partnerId,
+            body: "Another msg",
+            model: "discuss.channel",
+            message_type: "comment",
+            res_id: channelId,
+        },
+    ]);
+    await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-Message-actions [title='Expand']");
+    await contains(".o-dropdown-item:contains('Reply')");
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message:contains('Test message from B') button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer.o-focused .o-mail-Composer-input", { value: "@Partner B " });
+    await click(".o-mail-Composer button", { text: "Log" });
+    await contains(".o-mail-Message a.o_mail_redirect", { text: "@Partner B" });
+    await contains(".o-mail-Message:contains('@Partner B') button[title='Edit']");
+    await contains(".o-mail-Message:contains('@Partner B') button[title='Reply']", { count: 0 });
+    await click(".o-mail-Message:contains('@Partner B') .o-mail-Message-actions [title='Expand']");
```

Comment:
`.o-mail-Message-actions` can be removed (`Message:contains() [title='Expand']` is enough.